stacks/meteor/setup.sh

```diff
@@ -3,7 +3,7 @@ set -euo pipefail
 
 cd /opt/
 
-PACKAGE=meteor-spk-0.1.4
+PACKAGE=meteor-spk-0.1.6
```

Comment:
@kentonv says this needs to be 0.1.7 because he flubbed the first release - I'll take care of this.